### PR TITLE
Staging to main: LF announcement and GRU update

### PR DESCRIPTION
**Pull request by miguelgfierro**
### Checklist:

- [ ] I have followed the [contribution guidelines](../CONTRIBUTING.md) and code style for this project.
- [ ] I have added tests covering my contributions.
- [ ] I have updated the documentation accordingly.
- [ ] This PR is being made to `staging branch` and not to `main branch`.

### CODE REVIEW

**Review by anargyri — APPROVED** [review]

**Review by loomlike — APPROVED** [review]

File: NEWS.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
+<!--
+Copyright (c) Recommenders contributors.
+Licensed under the MIT License.
+-->
+
 # What's New
 
+## Update October 3, 2023
+
+We are pleased to announced that we have joined the [Linux Foundation of AI and Data](https://lfaidata.foundation/) (LF AI & Data). With this move, our objective is to keep building an ecosystem to sustain open source innovation and collaboration in recommendation systems.
```
[the remark this answers is not in the document]
Comment (wutaomsft) [reply NEWS.md:10]: @miguelgfierro did you see these comments?

**Review by wutaomsft — DISMISSED** [review]

File: NEWS.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (miguelgfierro) [reply NEWS.md:10]: @wutaomsft new PR: https://github.com/recommenders-team/recommenders/pull/2008 please review

File: README.md [hunk 2]
```diff
@@ -1,20 +1,25 @@
...
 
 [![Documentation Status](https://readthedocs.org/projects/microsoft-recommenders/badge/?version=latest)](https://microsoft-recommenders.readthedocs.io/en/latest/?badge=latest)
 
 <img src="https://raw.githubusercontent.com/recommenders-team/artwork/main/color/recommenders_color.svg" width="800">
 
-## What's New (August, 2023)
+## What's New (October, 2023)
 
-We moved to a new organization! Now to access the repo, instead of going to https://github.com/microsoft/recommenders, you need to go to https://github.com/recommenders-team/recommenders. The old URL will still resolve to the new one, but we recommend that you update your bookmarks.
+We are pleased to announced that we have joined the [Linux Foundation of AI and Data](https://lfaidata.foundation/) (LF AI & Data). With this move, our objective is to keep building an ecosystem to sustain open source innovation and collaboration in recommendation systems.
```
[the remark this answers is not in the document]
Comment (miguelgfierro) [reply README.md:14]: PR here: https://github.com/recommenders-team/recommenders/pull/2008

**Comment by miguelgfierro** [comment]
@anargyri @wutaomsft @loomlike @SimonYansenZhao @yueguoguo the official announcement will be Tuesday, 10/10 at 9:00 AM PST. 

There will be a blog post from MS and another from the LF

**Comment by yueguoguo** [comment]
Thanks for driving this @miguelgfierro and team!

**Comment by miguelgfierro** [comment]
Added ideas for collaborators https://github.com/recommenders-team/recommenders/discussions/2011

**Pull request merged**